test/common/fixtures/plugins/coverage/public/plugin.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Plugin } from 'kibana/server';
+
+interface WindowWithCoverage extends Window {
+  __coverage__: any;
+  flushCoverageToLog: any;
+}
+
+export class CodeCoverageReportingPlugin implements Plugin {
+  constructor() {}
+
+  public start() {}
+
+  public setup() {
+    (window as WindowWithCoverage & typeof globalThis).flushCoverageToLog = function () {
```

Comment:
I think we can clean this up by creating a local variable with the types assigned once and use that local variable rather than re-casting window over and over.

Reply:
@spalger @wayneseymour what do you think about 38163d2f52e56eb73dae834bafa72f671826c256

```
declare global {
  interface Window {
    __coverage__: any;
    flushCoverageToLog: any;
  }
}
```